src/domain.ts

```diff
@@ -176,3 +176,6 @@ export enum RunModelType {
 export interface EnvironmentVariables {
   [key: string]: string | undefined;
 }
+
+export const DefaultQueryTemplate =
```

Comment:
If consts use all caps. This looks like a class

Reply:
renamed.